timesys/high-level-triggers.py

```diff
@@ -1812,104 +1833,44 @@
             },
         'channels': (
             'SI-01M1:DI-BPM:CRT4', 'SI-01M2:DI-BPM:CRT4',
-            'SI-01C1:DI-BPM-1:CRT4', 'SI-01C1:DI-BPM-2:CRT4',
```

Comment:
why do we removed these BPMs? maybe we need to keep those that are related to BC

Reply:
I didn't understand the advantage of keeping all the other BPM CRT4 output without a known configuration (forcing them to be disabled, for example)

Reply:
Yeah, maybe it would be useful to have a high level trigger with the complement of these connections, to guarantee they are disabled, right?

Reply:
Yes, maybe the OutOrbIntlk high level trigger could be this one?

Reply:
Sounds good. However, in This case we need to be careful not to defining the same trigger output from the AMCFPGAEVR to both triggers. @ericonr, I think we could or come back the triggers from the same crate to this trigger here, or not include them in the new trigger, ok?

Reply:
hi @ericonr, I think I misunderstood the code. When I made the comment above I hadn't seen you had already created the other trigger I mentioned. I took a look again and it seems correct.